### ISSUE TITLE
hospital-admissions and doctor-visits available over different date ranges at different geo levels

### ISSUE THREAD

**Issue by capnrefsmmat** [cmu-delphi/covidcast-indicators#1038]
**Actual Behavior:**

```
> covidcast_meta() %>% filter(data_source == "hospital-admissions", signal == "smoothed_adj_covid19_from_claims") %>% select(geo_type, min_time, max_time)
A `covidcast_meta` data frame with 6 rows and 3 columns.

Number of data sources : 0
Number of signals      : 0

  geo_type   min_time   max_time
1   county 2020-02-01 2021-04-30
2      hhs 2020-10-04 2021-04-30
3      hrr 2020-02-01 2021-04-30
4      msa 2020-02-01 2021-04-30
5   nation 2020-10-04 2021-04-30
6    state 2020-02-01 2021-04-30
> covidcast_meta() %>% filter(data_source == "doctor-visits", signal == "smoothed_adj_cli") %>% select(geo_type, min_time, max_time)
A `covidcast_meta` data frame with 6 rows and 3 columns.

Number of data sources : 0
Number of signals      : 0

  geo_type   min_time   max_time
1   county 2020-02-01 2021-04-30
2      hhs 2021-01-05 2021-04-30
3      hrr 2020-02-01 2021-04-30
4      msa 2020-02-01 2021-04-30
5   nation 2021-01-05 2021-04-30
6    state 2020-02-01 2021-04-30
```

**Expected behavior**

Ideally, all geo levels would be available over the same time range, so e.g. if I want a nation-level analysis, I don't have to bodge it together myself from state-level signals.

**Comment by capnrefsmmat** [comment]
(Also the epidata docs should be updated to list the additional geo-levels for these sources, once they're backfilled)

**Comment by krivard** [comment]
@mariajahja how awful to do would this be? we don't need county/hrr/msa/state; is it possible to 1-off just the two missing geo levels?

**Comment by mariajahja** [comment]
@krivard Yes, we can do just national/hhs. Same procedure as before (tarball), or any changes that I need to know about?

**Comment by krivard** [comment]
Nope, same procedure is perfect. Thanks!

**Comment by mariajahja** [comment]
Sorry for the delay, the various formats/drop locations made it tricky to automate. The regen issues are in `/common/regen_hsp_nat_hhs/`. I did various spot checks using other regions, and they seem to line up. 

Notes: 
There may be additional issues for nat/hhs which are not stored in the API -- this is because historically those drops arrived late and we didn't retrospectively run the pipeline. There are also a few early 2020 issues not included for nat/hhs, these issues were manually run during that time due to various format changes that aren't compatible with the current pipeline. I ran a couple batches where the manual changes lasted some time, but the one-offs would take a lot more time. 

### Hospital admissions
_State, hrr, msa, county_
First issue: 2020-06-21
First date: 2020-02-01

_Nation/hhs_
First issue: 2020-12-10
First date: 2020-10-04

**Generated**:
   - Issue 2020-06-21, for dates [2020-02-01, 2020-06-18]
   - Issues (2020-06-21, 2021-01-10], for dates [issue_date - 70, issue_date - 3]
 ___
### Doctor visits
_State, hrr, msa, county_
First issue: 2020-04-29
First date: 2020-02-01

_Nation/hhs_
First issue: 2021-03-19
First date: 2021-01-05

**Generated**:
   - Issue 2020-06-06, for dates [2020-02-01, 2020-06-03]
   - Issues (2020-06-06, 2021-03-16], for dates [issue_date - 70, issue_date - 3]

**Comment by krivard** [comment]
🙌  thank you! flagging these for @dshemetov to log as missing data scenarios we may not have imagined in our first pass of the nans implementation.

i'll add these to the batch ingestion pool and post an update when they've been pulled in.

**Comment by ryantibs** [comment]
@krivard Is there any update on when this fix will go live? 

I was just playing around with the current dashboard and noticed this myself (screenshots below) and before creating an issue I searched around and found this one.

![Screen Shot 2021-09-27 at 2 31 40 PM](https://user-images.githubusercontent.com/732541/134965723-260c593a-45b5-45cd-af8b-bd87013d52de.png)
![Screen Shot 2021-09-27 at 2 31 46 PM](https://user-images.githubusercontent.com/732541/134965725-11fd8887-b741-41be-b831-97eb2a4f891a.png)

**Comment by krivard** [comment]
Good catch! Looks like this got sidelined for more urgent batch uploads and was never restarted. We should be able to have these up by the end of the week.

**Comment by dshemetov** [comment]
This one slipped past me. So my understanding of what happened here was: 
- we manually backissued data 
- bypassed the regular pipeline 
- backissued data was therefore never aggregated to larger geos

Is that right?
